java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1751,14 +1751,13 @@ public T zrangeWithScores(@NonNull String key, @NonNull ScoredRangeQuery rangeQu
      * @apiNote <code>BZMPOP</code> is a client blocking command, see <a
      *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
      *     Commands</a> for more details and best practices.
-     * @param timeout The number of seconds to wait for a blocking <code>BZMPOP</code> operation to
-     *     complete. A value of <code>0</code> will block indefinitely.
+     * @param timeout The number of seconds to wait for a blocking operation to complete. A value of
+     *     <code>0</code> will block indefinitely.
      * @param keys The keys of the sorted sets.
      * @param modifier The element pop criteria.
-     * @return Command Response - A two-element array with the first element being the name of the key
-     *     from which elements were popped, and the second element is a map of the popped element. The
-     *     only entry in the map contains the popped member and its corresponding score.<br>
-     *     The data could be interpreted in format as <code>[key, element, score]</code>.<br>
+     * @return Command Response - A two-element <code>array</code> containing the key name of the set
+     *     from which an element was popped, and a member-score <code>Map</code>.<br>
```

Comment:
Should this also be "were popped" like in the other bzmpop function below?
```suggestion
     *     from which an elements were popped, and a member-score <code>Map</code>.<br>
```

Reply:
No - it returns only one element